### ISSUE TITLE
what's the meaning of 'label' parameter in soft_cross_entropy() function?

### ISSUE THREAD

**Issue by bmxbmx3** [open-mmlab/mmclassification#805]
### Checklist
- I have searched related issues but cannot get the expected help.
- I have read related documents and don't know what to do.

### Describe the question you meet

I have learned the formula of cross entropy:
![image](https://user-images.githubusercontent.com/33427015/165672278-24c3e8d7-a84c-4119-8c86-dcfed20d9f42.png)

after I also learned the concept of softmax, I have checked the code of soft_cross_entropy() in file ( 'mmclassification/mmcls/models/losses/cross_entropy_loss.py') :
![image](https://user-images.githubusercontent.com/33427015/165672676-e8b38cd6-d3c0-4d9b-8758-a67506f52d18.png)

I don't understand why the gt label of the prediction is shape (N, C) ? I guess it's one-hot encoding, am I right?

### Post related information
1. The output of `pip list | grep "mmcv\|mmcls\|^torch"`
[here]
2. Your config file if you modified it or created a new one.
```python
[here]
```
3. Your train log file if you meet the problem during training.
[here]
4. Other code you modified in the `mmcls` folder.
[here]

**Comment by Ezra-Yu** [comment]
As the [desc](https://github.com/open-mmlab/mmclassification/blob/6beac5060f0c1d57141642dba77c19c9ed6b3873/mmcls/models/losses/cross_entropy_loss.py#L54) saying, soft_cross_entropy is designed when using mixup or label-smoothing, which label is not an integer. In some cases, such as mixup in [batch_aug](https://mmclassification.readthedocs.io/en/latest/api/models.utils.augment.html), the label is `[0, 0, ....0.9, ....., 0.1, .....]` of one picture, the batch label is shape of (N, C)

**Comment by Ezra-Yu** [comment]
Does this answer solve your question?  I will close this issue later.
